src/content/docs/infrastructure/host-integrations/host-integrations-list/lighttpd-integration.mdx

```diff
@@ -102,7 +102,7 @@ Path:
 
 You can see a sample configuration file [here](https://github.com/newrelic/nri-flex/blob/master/examples/lighttpd-http-example.yml).
 
-It's important to not that you can replace `base_url` and `url` with your application url (Please refer to url documentation for more understanding) and also change url in the `commands` attribute. You can also convert the metrics format which you get through url into regex and add expressions in regex_matches. EVENT_TYPE is also adjustable. 
+It's important to not that you can replace `base_url` and `url` with your application url (Please refer to [nri-flex url documentation](https://github.com/newrelic/nri-flex/blob/master/docs/apis/url.md) for more understanding) and also change url in the `commands` attribute. You can also convert the metrics format which you get through url into regex and add expressions in regex_matches. EVENT_TYPE is also adjustable. 
```

Comment:
@pkosanam thanks for adding this URL. Since we're looking at this line, I'm not really sure what it is saying:

* Could you clarify what the first part of the sentence means: "It's important to not that you can replace." Is this supposed to say this: "Do not replace `base_url` and `url` with your application URL?
* What is the meaning of the second part of the same sentence and how does it relate to the first part: "and also change url in the `commands` attribute." Could you clarify the whole sentence?

I'd also recommend that we use back ticks around key words such ad regex_matches and EVENT_TYPE.

Finally, could you sign the CLA agreement so we can publish this?

Reply:
Thank you @rhetoric101 , for your comment.
I have updated the text accordingly.